cmd/openshift/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-python-task-snippet.yaml

```diff
@@ -2,31 +2,31 @@
 apiVersion: console.openshift.io/v1
 kind: ConsoleYAMLSample
 metadata:
-  name: s2i-python-3-snippet
+  name: s2i-python-snippet
 spec:
   targetResource:
     apiVersion: tekton.dev/v1beta1
     kind: Pipeline
-  title: S2I-Python-3 Workspace based Task
+  title: S2I-Python Workspace based Task
   description: |
-    An s2i task to build python 3 based source.
+    An s2i task to build python based source.
     Source-to-Image (S2I) is a toolkit and workflow for building reproducible container images from source code.
     S2I produces images by injecting source code into a base S2I container image and letting the container prepare that source code for execution.
     The base S2I container images contains the language runtime and build tools needed for building and running the source code.
   snippet: true
   yaml: |
     - name: build
       taskRef:
-        name: s2i-python-3
+        name: s2i-python
         kind: ClusterTask
       workspaces:
         - name: source
           workspace: workspace
       params:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
-        - name: MINOR_VERSION
-          value: "6"
+        - name: VERSION
+          value: "3.8"
```

Comment:
why not the same version as in clustertask, i.e. `3.8-ubi8`?

Reply:
as these are snippets I thought to give a different version. 
But we can update it to the same as the clustertasks.